src/harness/fourslashImpl.ts

```diff
@@ -3576,6 +3576,72 @@ export class TestState {
         }
     }
 
+    public baselineLinkedEditing(): void {
+        const baselineFile = this.getBaselineFileNameForContainingTestFile(".linkedEditing.txt");
+        const files = this.testData.files;
+
+        let baselineContent = "";
+        let offset = 0;
+        files.forEach(f => {
+            const result = getLinkedEditingBaselineWorker(f, offset, this.languageService);
+            baselineContent += result.baselineContent + `\n\n\n`;
+            offset = result.offset;
+        });

```

Comment:
nit - just use a for-of loop

```suggestion
        for (const f of files) {
            const result = getLinkedEditingBaselineWorker(f, offset, this.languageService);
            baselineContent += result.baselineContent + `\n\n\n`;
            offset = result.offset;
        }
```